src/taipy/core/common/_entity_ids.py

```diff
@@ -33,3 +34,9 @@ def __add__(self, other: _EntityIds):
     def __iadd__(self, other: _EntityIds):
         self.__add__(other)
         return self
+
+    @property
+    def _union_all(self) -> Set:
```

Comment:
Not sure what this function is for?

Reply:
This is to return a set for all the entity ids in the `EntityIds` object itself. It is not used after I have made some changes with the export implementation, but you can keep it if it can be useful in the future.